core/controller/src/main/scala/whisk/core/controller/RestAPIs.scala

```diff
@@ -173,8 +173,6 @@ protected[controller] class RestAPIVersion(apipath: String, apiversion: String)(
                     webexp.routes()
                 } ~ {
                     swaggerRoutes
-                } ~ options {
```

Comment:
this also removes it from the main like API (which I agree is silly to preflight since it's authenticated) but that's a breaking change (vs experimental).